bin/artisan

```diff
@@ -0,0 +1,31 @@
+#!/usr/bin/env php
```

Comment:
not sure about putting it here. This file doesn't exist for Composer so the consumers will don't mind it. It would maybe make more sense to put it with the Laravel app, but it becomes very annoying to use it (very long path) and we are the ones using it, not the consumers.